hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/workload/selective/BalancingJob.java

```diff
@@ -51,12 +51,12 @@ public void run() {
         try {
             if (consumersRegistry.isLeader()) {
                 try (Timer.Context ctx = metrics.consumersWorkloadRebalanceDurationTimer(kafkaCluster).time()) {
-                    logger.info("Initializing workload balance.");
+//                    logger.info("Initializing workload balance.");
```

Comment:
I forgot to uncomment these ones. I was debugging a problem and these messages were spamming entire logs.